wp-content/plugins/pdf-creator-lite/scripts/build-pdf-admin.php

```diff
@@ -208,7 +226,13 @@ function ( $matches ) {
 					
 					$pdf->AddPage();				
 					$pdf->Bookmark( $title, $depth, 0, '', 'B', array($link_rgb['red'], $link_rgb['green'], $link_rgb['blue']), 0, '#TOC' );
-					
+
+					//XXX resolve bug with <p> tags within a table
```

Comment:
yes, i think so, but I only did minor tests. All tables from the Augsburg site looked fine.